### PR TITLE
Update Parser Input to new command format

### PR DESCRIPTION
**Pull request by s-peiran**
### Description
Standardize on how user should enter their respective command in parser

### Related Issue(s)
[#74](https://github.com/AY2324S1-CS2103-W14-2/tp/issues/74)

### Checklist
[Make sure to check the boxes that apply. If an item is not applicable, you can remove it.]

- [x] Modify parser based on how user should be using it
- [x] Update user guide for command examples
- [x] Create dummy class for EditMeetingCommand and DeleteMeetingCommand

### Screenshots (if applicable)
[Include any relevant screenshots to help reviewers understand the changes.]

### CODE REVIEW

File: bin/main/seedu/address/AppParameters.class
Comment (chiayunrong) [inline bin/main/seedu/address/AppParameters.class:1]: I don't think this entire directory is supposed to be added?

File: docs/UserGuide.md [hunk 1]
```diff
@@ -33,28 +31,29 @@ Notenote provide tools for organizing and categorizing contacts in a systematic
 
 - **What it does**: Adds a new contact to the list.
 
-- **Command Format**: `contact name/CONTACT_NAME`
+- **Command Format**: `add contact -n CONTACT_NAME -p PHONE_NUMBER -e EMAIL_ADDRESS -a RESIDENTIAL_ADDRESS [-t TAGS]`
```
Comment (chiayunrong) [inline docs/UserGuide.md:34]: I like the new format :D

File: src/main/java/seedu/address/logic/parser/CliSyntax.java [hunk 2]
```diff
@@ -6,14 +6,15 @@
...
-    public static final Prefix PREFIX_TIME = new Prefix("time/");
-    public static final Prefix PREFIX_PLACE = new Prefix("place/");
-    public static final Prefix PREFIX_DESCRIPTION = new Prefix("desc/");
-
+    public static final Prefix PREFIX_NAME = new Prefix("-n");
+    public static final Prefix PREFIX_PHONE = new Prefix("-p");
+    public static final Prefix PREFIX_EMAIL = new Prefix("-e");
+    public static final Prefix PREFIX_ADDRESS = new Prefix("-a");
+    public static final Prefix PREFIX_TAG = new Prefix("-t");
+    public static final Prefix PREFIX_TITLE = new Prefix("-title");
```
Comment (chiayunrong) [inline src/main/java/seedu/address/logic/parser/CliSyntax.java:14]: Should we change the following prefixes to a 1 letter prefix?

**Review by chiayunrong — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/address/logic/parser/CliSyntax.java (hunk 2 above)
Comment (s-peiran) [reply src/main/java/seedu/address/logic/parser/CliSyntax.java:14]: Lets look into it again in the future,
but currently for contacts all is 1 letter prefix,
and for meeting all is 1 word prefix
so it is somewhat standardized for the moment.

**Review by jonyeokj — APPROVED** [review]

**Pull request merged**